### DISCUSSION TITLE
How to introduce a delay for registers popup ?

### DISCUSSION THREAD

**Discussion by elankath** [NvChad/NvChad#1922] [Q&A] [answered]
Hi all, The registers popup when one presses `'` or `"` is very distracting especially when editing. One one is doing quick copy-paste operations, this popup flash just interferes with work.  Is there a way to increase the delay for this registers popup ? 

All help appreciated!

**Comment by siduck** [comment, marked as answer]
our timeoutlent is 400 , 

```lua
vim.opt.timeoutlen = 800
```

add in custom/init.lua

**Reply by elankath** [reply]
thank you, but I guess this does it globally and not just for registers.

**Reply by siduck** [reply]
@elankath  I think whichkey takes value from that option , check whichkey issues page or docs
